namespaces/strings/git/src/lib.rs

```diff
@@ -80,3 +80,30 @@ pub extern "C" fn staged_count() -> *mut i8 {
                   STATUS_INDEX_RENAMED | STATUS_INDEX_TYPECHANGE;
     count_statuses(staged)
 }
+
+fn count_revs_between_branches(from: &str, to: &str) -> *mut i8 {
+    use git2::*;
+
+    if let Ok(dir) = current_dir() {
+        if let Ok(repo) = Repository::open(dir) {
+            if let Ok(mut revwalk) = repo.revwalk() {
+                if let Ok(_) = revwalk.push_range(&format!("{}..{}", from, to)) {
+                    let count = revwalk.count();
+                    return build_return_value(format!("{}", count));
+                }
+            }
+        }
+    }
```

Comment:
If you ever want to know how to do the above without the if lets, you can do it like this:

```rust
current_dir()
    .and_then(|dir| Repository::open(dir).ok())
    .and_then(|repo| repo.revwalk().ok())
    .and_then(|mut revwalk| {
        revwalk.push_range(&format!("{}..{}", from, to))
            .map(|_| build_return_value(format!("{}", revwalk.count))
            .ok()
    })
    .unwrap_or(ptr::null_mut())
```